### ISSUE TITLE
Make <H> tags on Community and homepage clickable

### ISSUE THREAD

**Issue by logankilpatrick** [JuliaLang/www.julialang.org#868]
Hey @tlienart, how can we make the H tag's on the various pages clickable?

**Comment by zdroid** [comment]
If the page is in Markdown and you use the Markdown syntax for headings, it's done automatically. (Look at https://julialang.org/downloads/#older_releases for example)

**Comment by logankilpatrick** [comment]
Yes, but for the pages where we embed the HTML in markdown, those don’t work.

**Comment by logankilpatrick** [comment]
Ah, okay thanks!

**Comment by zdroid** [comment]
I guess this should be closed if @tlienart's suggestion resolves your issue, right?

**Comment by zdroid** [comment]
@logankilpatrick I guess this should be closed if @tlienart's suggestion resolves your issue, right?
